### PR TITLE
[WEB-5641] show the correct path

### PR DESCRIPTION
**Pull request by VeranikaVasilchykava**
https://bowery.atlassian.net/browse/WEB-5641
I also updated initialWhiteList, set 5min to inactivity interval.

### CODE REVIEW

File: src/scripts/background.js [hunk 1]
```diff
@@ -266,15 +266,27 @@ const extractTime = (start, end) => {
     }
 }
 
+const cleanIntervals = (intervals = []) => {
+    return intervals.filter(interval => {
+        const intervalStart = interval.split('-')[0];
+        const intervalEnd = interval.split('-')[1];
```
Comment (mrromro) [inline src/scripts/background.js]: You can use destructuring like `const [intervalStart, intervalEnd] = interval.split('-')`.

File: src/scripts/background.js [hunk 2]
```diff
@@ -266,15 +266,27 @@ const extractTime = (start, end) => {
...
+        const startIndex = intervalStart.lastIndexOf(':');
+        const endIndex = intervalEnd.lastIndexOf(':');
+        const intervalStartWithoutMs = intervalStart.slice(0, startIndex);
+        const intervalEndWithoutMs = intervalEnd.slice(0, endIndex);
```
Comment (mrromro) [inline src/scripts/background.js:275]: By the way, I like your descriptive way of variable naming.

File: src/scripts/background.js [hunk 3]
```diff
@@ -266,15 +266,27 @@ const extractTime = (start, end) => {
     }
 }
 
+const cleanIntervals = (intervals = []) => {
```
Comment (mrromro) [inline src/scripts/background.js]: To avoid confusing with intervals and timeout apis you can name this function `extractIntervals`.

**Review by mrromro — APPROVED** [review]

File: src/scripts/background.js (hunk 1 above)
Comment (VeranikaVasilchykava) [reply src/scripts/background.js]: done)

File: src/scripts/background.js (hunk 3 above)
Comment (VeranikaVasilchykava) [reply src/scripts/background.js]: good point, thank you!

**Pull request merged**